manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "Is Eddie Here?",
+  "icons": [
+    {
+      "src": "icon/icon-0-75x.png",
```

Comment:
icon/icon* doesn't make a lot of sense to me. why would something in icon/ not be icon-related?